pytest_watch/command.py

```diff
@@ -87,23 +91,24 @@ def main(argv=None):
     if args['--pdb']:
         pytest_args.append('--pdb')
 
-    # Parse extensions
+    # Parse extensions [default: .py]
     if args['--ext'] == '*':
         extensions = ALL_EXTENSIONS
     elif args['--ext']:
         extensions = [('.' if not e.startswith('.') else '') + e
                       for e in args['--ext'].split(',')]
-    else:
-        extensions = None
 
     # Parse numeric arguments
     spool = args['--spool']
-    if spool is not None:
-        try:
-            spool = int(spool)
-        except ValueError:
-            sys.stderr.write('Error: Spool must be an integer.\n')
-            return 2
+    try:
+        spool = int(spool)
+    except ValueError:
+        sys.stderr.write('Error: Spool (--spool %s) must be an integer.\n' % spool)
```

Comment:
Two things.

1. I used to willfully break PEP8's "80 character limit" rule all the time. With side-by-side comparisons and 3-way merges, I've since come to appreciate this style rule. Could you do the same here?

2. I thought I read a while back that Python favored `.format` over `%` these days. A quick Google search yields no results on this. But since I've since switched to using `.format` everywhere, could you do the same here for consistency? A case could be made later to go back to `%` everywhere.

This should do it:

```
        sys.stderr.write(
            'Error: Spool (--spool {}) must be an integer.\n'.format(spool))
```

Or perhaps:

```
        print(
            'Error: Spool (--spool {}) must be an integer.\n'.format(spool),
            file=sys.stderr)
```